charmcraft/templates/init-simple/tox.ini.j2

```diff
@@ -40,7 +40,7 @@ deps =
 commands =
     # uncomment the following line if this charm owns a lib
     # codespell {[vars]lib_path}
-    codespell {toxinidir}/. \
+    codespell {toxinidir} \
```

Comment:
This probably needs to be changed in `init-machine/tox.ini.j2` and `init-kubernetes/tox.ini.j2` as well.

Reply:
Pretty sure they're just symlinks to here, no?